unique_toolkit/unique_toolkit/language_model/service.py

```diff
@@ -321,3 +330,46 @@ def _add_tools_to_options(
                 for tool in tools
             ]
         return options
+
+    @staticmethod
```

Comment:
Since this is a static method? wouldn't it be better as a utility method and not prefixed with `_` which signifies that it is private?